src/app.ts

```diff
@@ -153,6 +160,14 @@ async function app() {
   results.forEach(r => console.log(r));
 
   if (program.fshingTrip) {
+    if (fileType === 'xml-only') {
```

Comment:
If we do treat this case-insensitive elsewhere, we need to be case-insensitive everywhere else too; or normalize the `fileType` when we assign it by doing a `toLowerCase()`.